### PR TITLE
ZBUG-3378: Add makefile to create package for zimlet

### PR DESCRIPTION
**Pull request by gautamdg**
Add makefile to create package for zimlet

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,41 @@
+########################################################################################################
+
+SHELL=bash
+PACKAGE_XML = ./js/com_zimbra_clientuploader.xml
+NAME = zimbra-zimlet-clientuploader
```
Comment (silentsakky) [inline Makefile]:
```suggestion
NAME = zimbra-zimlet-admin-clientuploader
```

File: scripts/postinst.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+echo "*** Configuring zimbra-zimlet-clientuploader ***"
```
Comment (silentsakky) [inline scripts/postinst.sh]: name change

Comment (gautamdg) [reply scripts/postinst.sh]: Done

File: Makefile (hunk 1 above)
Comment (gautamdg) [reply Makefile]: Done

**Review by umagmrit — APPROVED** [review]

**Pull request merged**